src/main/java/org/elasticsearch/action/support/broadcast/TransportBroadcastOperationAction.java

```diff
@@ -60,12 +59,16 @@ protected TransportBroadcastOperationAction(Settings settings, String actionName
         this.clusterService = clusterService;
         this.transportService = transportService;
         this.threadPool = threadPool;
-        this.transportShardAction = actionName + "[s]";
+        this.transportShardAction = actionName + getShardOperationNameSuffix();
```

Comment:
If we need this (after the removal of `SyncedFlushActionTests`), I think it makes more sense to make a method called shardAction(String baseAction) which generates the action name, rather then exposing the prefix and combining it on the outside.